site/doc/1.11/journal.md

```diff
@@ -1242,9 +1296,14 @@ something.):
 ```
 
 The posting amounts can be of the form `*N`, which means "the amount of
-the matched transaction's first posting, multiplied by N". They can also
-be ordinary fixed amounts. Fixed amounts with no commodity symbol will
-be given the same commodity as the matched transaction's first posting.
+the matching posting(s), multiplied by N". They can also be ordinary
+fixed amounts (which are inserted unchanged) or [amount
+expressions](#amount-expressions) headed by either (only the
+multiplication to the left of the first addition or subtraction will be
+applied to the posting value).
+
+Any postings in real transactions which attempt to use this
+multiplication-headed form will ignore the value instead.
```

Comment:
I didn't understand "headed by" onward exactly. Shall we can simplify again, and delete those lines ? Ie we will accept an amount, an amount expression, or a numeric multiplier (`*N`). (Adding support for `\N`, `+N`, `-N` might be worthwhile later ?)

Reply:
Blame a couple years of linguistics.  That's the passage that was primary in the "first draft" naming of the commit.  I'll revisit this.

Reply:
On the "those can wait", I might actually put the first step of the PR together without multiplication entirely.  Addition and subtraction are a much more contained change, and seem to generate much less discussion (while being more frequently requested).  We should definitely keep working out the questions around multiplication, but splitting things further like that means there's less pressure to resolve it so we can find something we're all happier with.